frontend/src/main/java/com/google/tripmeout/frontend/PlaceVisitModel.java

```diff
@@ -11,17 +11,20 @@ public static Builder builder() {
     return new AutoValue_PlaceVisitModel.Builder().setUserMark(UserMark.UNKNOWN);
   }
 
+  @Nullable public abstract String uuid();
   public abstract String placesApiPlaceId();
-  @Nullable public abstract String placeName();
-  public abstract String tripId();
+  @Nullable public abstract String tripId();
   public abstract UserMark userMark();
+  // TO-DO: decide if we still need/want these fields
```

Comment:
We'll want to drop all the place-specific metadata that we get from the place details API (latitude and longitude). This prevents issues with stale data as we discussed offline.

It might make sense to keep some sort of friendly name that the user can specify. The client can always query the place details API for this info, but that won't include their own custom name for the visit itself.